src/overlays/ovl_i4/fox_sz.c

```diff
@@ -120,132 +129,142 @@ void SectorZ_Missile_Update(Actor* this) {
             break;
     }
 
-    sp2C = gBosses[0].obj.pos.x - this->obj.pos.x;
-
-    if (0) {} //! FAKE
-
-    sp24 = gBosses[0].obj.pos.z - this->obj.pos.z;
-
-    SIN_DEG(gGameFrameCount); // Leftover from a test or commented out code?
-
-    if (sp2C) {} //! FAKE
-    if (sp24) {} //! FAKE
+    /* Leftover from a test or commented out code? */
+    /* =========================================== */
+    x = gBosses[0].obj.pos.x - this->obj.pos.x;
+    y = gBosses[0].obj.pos.y - this->obj.pos.y; // Optimized out?
+    if (1) {} //! FAKE
+    z = gBosses[0].obj.pos.z - this->obj.pos.z;
+    SIN_DEG(gGameFrameCount);
+    if (x && z) {} //! FAKE
+    /* =========================================== */
 
     if (this->aiType < 100) {
-        if (sp24) {}                                    //! FAKE
-        sp2C = fabsf(this->fwork[4] - this->obj.pos.x); //! FAKE
-        sp20 = SIN_DEG((this->index * 45) + gGameFrameCount) * 5000.0f;
-        sp1C = COS_DEG((this->index * 45) + (gGameFrameCount * 2)) * 5000.0f;
+        if (z) {} //! FAKE
+        z = fabsf(this->fwork[6] - this->obj.pos.z); //! FAKE
+
+        // Escorts wooble movement
+        xPitch = SIN_DEG((this->index * 45) + gGameFrameCount) * 5000.0f;
+        yPitch = COS_DEG((this->index * 45) + (gGameFrameCount * 2)) * 5000.0f;
     } else {
-        sp20 = sp1C = 0.0f;
+        // For missile
+        xPitch = yPitch = 0.0f;
     }
 
-    this->fwork[4] = gBosses->obj.pos.x + sp20 + 400.0f;
-    this->fwork[5] = gBosses->obj.pos.y + sp1C + 100.0f;
-    this->fwork[6] = gBosses->obj.pos.z;
+    this->fwork[MISSILE_TARGET_X] = gBosses[0].obj.pos.x + xPitch + 400.0f;
+    this->fwork[MISSILE_TARGET_Y] = gBosses[0].obj.pos.y + yPitch + 100.0f;
+    this->fwork[MISSILE_TARGET_Z] = gBosses[0].obj.pos.z;
```

Comment:
fwork 4-6 for all range actors appears to be a general purpose target position, so I don't think we need this to be missile-specific.